### ISSUE TITLE
Plugin Store

### ISSUE THREAD

**Issue by kevingroeger** [Splamy/TS3AudioBot#756]
**Is your feature request related to a problem? Please describe.**
This is not really related to a problem, other than me being too bad at C# to get how to write a plugin.

**Describe the solution you'd like**
I propose an addition to the website, where users can upload and share their plugins with others. As a webdev I would even offer my help. 

A nice side effect to this would be that the [unfinished plugin site on the wiki](https://github.com/Splamy/TS3AudioBot/wiki/Plugins) could benefit from this feature.

**Describe alternatives you've considered**
I mean, the only alternative for me is to not suck at C#.

**Comment by Splamy** [comment]
Yeah, I've also thought about that a lot, but creating a good plugin store from the ground isn't a easy task.
My current idea would be to leverage the nuget package manager by creating a plugin page on splamy.de which just redirects / proxies to normal packages on nuget.org/myget.org.
That would solve a lot of versioning/dependency resolving but it's still a lot of work to set all up.
